.travis.yml

```diff
@@ -252,6 +252,7 @@ before_script:
 
 script:
   - cat $HOME/.php-build/share/php-build/default_configure_options
+  - export PKG_CONFIG_PATH="$HOME/.phpenv/versions/$VERSION/lib/pkgconfig:$PKG_CONFIG_PATH"
```

Comment:
I am not entirely familiar with pkg_config; does this environment variable need to be present at run time?

Reply:
By runtime do you mean when PHP is executed? In that case no, pkg-config is only used by the build system.